cli/index.js

```diff
@@ -39,13 +43,355 @@ const signUp = (args) => {
+        .catch((error) => {
+            console.log(chalk.red.bold(`Could not update this account -> code: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
+        });
+}
+//get post command definition
+const getpost = (args) => {
+
+    console.log(chalk.yellow.bold("getting post..."));
+
+    const { post } = args;
+    
+        const getPostData = {
+            post
+        };
+
+    axios
+        .get("/posts", getPostData)
+        .then((res) => {
+            console.log(res.data);
+
+            const FBIdToken = `Bearer ${res.data.token}`;
+            // TODO: create a file in the user's machine to store the auth token
+            console.log(chalk.green.bold("you got the post successfully ✔️"));
+        })
+        .catch((error) => {
+            console.log(chalk.red.bold(`Could not obtain the post -> code: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
+        });
+}
+//create post command definition
+const createPost = (args) => {
+
+    const { post, canLike, canUnlike, likes, tag } = args;
+
+    const postData = {
+        post,
+        canLike,
+        canUnlike,
+        likes,
+        tag
+    };
+
+    console.log(chalk.yellow.bold("posting..."));
+
+    axios
+        .post("/posts", postData)
+        .then((res) => {
+            console.log(res.data);
+
+            const FBIdToken = `Bearer ${res.data.token}`;
+            // TODO: create a file in the user's machine to store the auth token
+            console.log(chalk.green.bold("post created successfully ✔️"));
+        })
+        .catch((error) => {
+            console.log(chalk.red.bold(`Could not create the post -> code: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
+        });
+}
+//update post command definition
+const updatePost = (args) => {
+
+    const { post, canLike, canUnlike, likes, tag } = args;
+
+    const putPostData = {
+        post,
+        canLike,
+        canUnlike,
+        likes,
+        tag
+    };
+
+    console.log(chalk.yellow.bold("editing the post..."));
+
+    axios
+        .put("/posts", putPostData)
+        .then((res) => {
+            console.log(res.data);
+
+            const FBIdToken = `Bearer ${res.data.token}`;
+            // TODO: create a file in the user's machine to store the auth token
+            console.log(chalk.green.bold("post updated successfully ✔️"));
+        })
+        .catch((error) => {
+            console.log(chalk.red.bold(`Could not update this post -> code: ${error.response.statusText}, message: ${JSON.stringify(error.response.data)}`));
+        });
+}
+
+/*  Delete post    */
+const deletePost = (args) => {
+
+    const {  } = args;
+
+    const deleteCo = {
+        post,
+        canLike,
+        canUnlike,
+        likes,
+        tag
+    };
+
+    console.log(chalk.yellow.bold("deleting..."));
+
+    axios
+        .post("/posts", deleteCo)
+        .then((res) => {

```

Comment:
the axios call to delete is not `axios.post`, is actually `axios.delete`

Reply:
done